projects/ngx-breadcrumb/tsconfig.lib.prod.json

```diff
@@ -5,6 +5,7 @@
     "declarationMap": false
   },
   "angularCompilerOptions": {
-    "enableIvy": false
+    "enableIvy": true,
```

Comment:
By https://v12.angular.io/guide/creating-libraries#publishing-libraries you should be able to omit `enableIvy` entirely, and just have `compilationMode`.

Reply:
`enableEvy` removed.